src/System.Net.Http.WinHttpHandler/src/System.Net.Http.WinHttpHandler.csproj

```diff
@@ -7,6 +7,8 @@
     <OutputType>Library</OutputType>
     <AssemblyName>System.Net.Http.WinHttpHandler</AssemblyName>
     <GeneratePlatformNotSupportedAssembly Condition="'$(TargetsUnix)' == 'true'">true</GeneratePlatformNotSupportedAssembly>
+    <!-- Although we have a NS configuration, the actual UWP implementation is a shim over WinRT: so just validate against OneCore -->
```

Comment:
Then we should consider removing the netstandard configuration and just having a netcoreapp and netfx configuration.

@ericstj/@terrajobst  - What is your thoughts about our packages that only support some platforms but the refs can be netstandard? Should we opt them into only the concrete platforms and not allow them to be consumed from netstandard? Or should we make them consumable by netstandard but let them fail to install in applications targeting other platforms?